config/initializers/cors.rb

```diff
@@ -7,10 +7,11 @@
 
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
   allow do
-    origins '*'
+    origins 'http://localhost:8080'
```

Comment:
Will need to update origins to be environment sensitive eg:
```suggestion
    origins ENV['localhost']
```

Reply:
See above suggestion